lichess-bot.py

```diff
@@ -685,24 +687,35 @@ def print_pgn_game_record(li: lichess.Lichess, config: Configuration, game: mode
     game_path = os.path.join(config.pgn_directory, game_file_name)
 
     lichess_game_record = chess.pgn.read_game(io.StringIO(li.get_game_pgn(game.id)))
+    if not isinstance(lichess_game_record, chess.pgn.Game):
```

Comment:
`lichess_game_record = chess.pgn.read_game(io.StringIO(li.get_game_pgn(game.id))) or chess.pgn.Game()`

This will negate the next `if not isistance`. The only way this line could fail is if there is a communication error with lichess, in which case the raised exception would end the function anyway.